packages/typespec-ts/src/modular/emitModels.ts

```diff
@@ -94,7 +94,13 @@ export function buildModels(
             type: propertyTypeName
           };
         })
-      });
+      };
+      model.type === "model"
+        ? model.parents?.forEach((p) =>
+            modelInterface.extends.push(getType(p).name)
+          )
+        : undefined;
+      modelsFile.addInterface(modelInterface);
```

Comment:
Fix model inheritence issue found in #1962

Reply:
maybe not in this pr, we could integrate inheritance case in cadl-ranch